docs/topics/circuit-breaker-mission-intro-paragraph.adoc

```diff
@@ -1,3 +1,6 @@
-Mission proficiency level: xref:proficiency_foundational[*{proficiency-foundational}*].
+Mission proficiency level: 
+//special case since topic is used by front end.
+ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
+ifndef::docs-topic[link:https://appdev.openshift.io/docs/wf-swarm-runtime.html#proficiency_levels[*Foundational*].]
```

Comment:
@edewit: No, you should not set `:docs-topic:` at all, setting this variable enables docs-only stuff. In the launchpad, you want it unset at all times.

Re foundational: In case of the launchpad line (L4), the string is hardcoded to `Foundational`. We only need it to use the same attribute as the docs line (L3), i. e. `{proficiency-foundational}`. No changes in the launchpad should be necessary as the value of this attribute is defined in `document-attributes.adoc`.

Re URL: If we want to link to the appropriate guide based on what runtime the user selected on the launchpad, we'll need you to set some attribute that'd help us generate the appropriate URL based on info in `document-attributes.adoc` (i. e. which runtime is being used). Having said that, if pointing to the WF Swarm book only is fine (the information in the appendix is identical in all guides after all), no changes need to be done.